public/app/features/integrated-alerting/components/Table/Table.test.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { dataQa } from '@percona/platform-core';
+import { Table } from './Table';
+
+const columns = [
+  {
+    Header: 'test col 1',
+    accessor: 'value',
+  },
+];
+
+const data = [
+  {
+    value: 'test value 1',
+  },
+  {
+    value: 'test value 2',
+  },
+];
+
+describe('Table', () => {
+  it('should render the table correctly', async () => {
+    const wrapper = mount(<Table data={data} columns={columns} />);
+
+    expect(wrapper.find(dataQa('table-thead')).find('tr')).toHaveLength(1);
+    expect(wrapper.find(dataQa('table-tbody')).find('tr')).toHaveLength(2);
+    expect(wrapper.find(dataQa('table-no-data'))).toHaveLength(0);
+  });
+
+  it('should render correctly when pending', async () => {
```

Comment:
```suggestion
  it('should render the loader when data fetch is pending', async () => {
```
Let's be explicit as to what the test is supposed to assert